src/components/SystemsPage/SystemsPage.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Table, TableBody } from '@material-ui/core';
+
+import { EmptyState, InfoCard } from '@backstage/core-components';
+
+import { useCortexApi } from '../../utils/hooks';
+import { SystemsPageRow } from './SystemsPageRow';
+import { useDetailCardStyles } from '../../styles/styles';
+
+import { GroupByOption } from '../../api/types';
+
+export const SystemsPage = () => {
+  const groupBy = GroupByOption.SERVICE_GROUP;
+  const { value: serviceScores } = useCortexApi(
+    api => api.getServiceScorecardScores(groupBy),
+    [groupBy],
+  );
+  const systemScores = serviceScores?.filter(serviceScore =>
+    serviceScore?.identifier?.includes('system:'),
+  );
+  const classes = useDetailCardStyles();
+  return (
+    <InfoCard title="Systems" className={classes.root}>
+      {serviceScores?.length === 0 ? (
+        <EmptyState missing="data" title="No systems found." />
+      ) : (
+        <Table>
+          <TableBody>
+            {systemScores?.map(serviceScore => (
```

Comment:
and we should invert this -- the systems tab will have top level list of scorecards, which consist of lists of services
the top level scorecards will be the averages of all the scores of the services underneath